src/AlgoliaSearch.tsx

```diff
@@ -9,10 +9,10 @@ type Props = Readonly<{
 
 export const AlgoliaSearch: FC<Props> = props => {
   const { hits, showMore, isLastPage } = useInfiniteHits<AlgoliaItem>()
-  const { isSearchStalled, query, refine, clear } = useSearchBox();
```

Comment:
I don't see this deprecated in https://www.algolia.com/doc/api-reference/widgets/search-box/react-hooks/#hook-api

Reply:
It seems they didn't propagate the depracation into the docs yet. It is present in the code jsdoc in the SDK. https://github.com/algolia/instantsearch/blob/master/packages/instantsearch.js/src/connectors/search-box/connectSearchBox.ts#L50. I didn't find the hook exactly, but this type should be the same as the hook props. If you open the downloaded sdk code in your favourite ide, you should find it easily on the hook parameters in the `.d.ts` file.